io.openems.edge.meter.sunspec/src/io/openems/edge/meter/sunspec/SunSpecMeters.java

```diff
@@ -0,0 +1,17 @@
+package io.openems.edge.meter.sunspec;
+
+public enum SunSpecMeters {
+	SOLAR_EDGE("SolarEdge", 2), //
+	ELGRIS_ZERO_EXPORT("Elgris Zero Export", 1), //
+	ELGRIS_SMART_METER("Elgris Smart Meter", 1), //
+	TQ_EM_420("TQ EM420", 1);
```

Comment:
This is similar to my critics in #1991. We should really create individual Meter components per hardware manufacturer/device. This makes it much more visible to OpenEMS users, which devices are supported and also allows for more specific adjustments per device down the line. This might not be needed so much for meters, but it's our experience e.g. with OCPP charging stations, etc.

See the implementation of SolarEdge Grid-Meter as an example (-> https://github.com/OpenEMS/openems/blob/develop/io.openems.edge.solaredge/src/io/openems/edge/solaredge/gridmeter/SolarEdgeGridMeter.java).

Reply:
I have created such components.